content/concepts/the-rippled-server/the-rippled-server.md

```diff
@@ -0,0 +1,31 @@
+---
+html: the-rippled-server.html
+funnel: Build
+doc_type: Concepts
+category: The rippled Server
+template: template-landing-children.html
+blurb: rippled is the core peer-to-peer server that manages the XRP Ledger. This section covers concepts that help you learn the "what" and "why" behind fundamental aspects of the rippled server.
+---
+# The `rippled` Server
+
+`rippled` is the core peer-to-peer server that manages the XRP Ledger. This section covers concepts that help you learn the "what" and "why" behind fundamental aspects of the `rippled` server.
+
+## Reasons to Run Your Own Server
+
+For lighter use cases and individual servers, you can often rely on free [public servers](https://xrpl.org/get-started-with-the-rippled-api.html#public-servers). However, the more serious your use of the XRP Ledger becomes, the more important it becomes to have your own infrastructure.
+
+There are lots of reasons you might want to run your own `rippled` server, but most of them can be summarized as: you can trust your own server, you have control over its workload, and you're not at the mercy of others to decide when and how you can access it. Of course, you must practice good network security to protect your server from malicious hackers.
+
+You need to trust the `rippled` you use. If you connect to a malicious server, there are many ways that it can take advantage of you or cause you to lose money. For example:
+
+* A malicious server could report that you were paid when no such payment was made.
+* It could selectively show or hide payment paths and currency exchange offers to guarantee its own profit while not providing you the best deal.
+* If you sent it your address's secret key, it could make arbitrary transactions on your behalf, and even transfer or destroy all the money your address holds.
+
+Additionally, running your own server gives you admin control over it, which allows you to run important admin-only and load-intensive commands. If you use a shared server, you have to worry about other users of the same server competing with you for the server's computing power. Many of the commands in the WebSocket API can put a lot of strain on the server, so `rippled` has the option to scale back its responses when it needs to. If you share a server with others, you may not always get the best results possible.
```

Comment:
> gives you admin control over it

Should that be "administrative control"?

Reply:
These docs use "admin" pretty consistently. I'd be willing to change my mind on the preferred term, but "admin" seems ubiquitous and unambiguous enough that I think it's better (when used as an adjective) than "administrative"